src/components/MuiVirtualizedTable/ColumnHeader.js

```diff
@@ -40,52 +41,50 @@ const useStyles = makeStyles((theme) => ({
     filterButton: {
         stroke: 'currentcolor',
     },
-    filterTooLossy: {
+    filterTooLossy: (theme) => ({
         stroke: theme.palette.secondary.main,
-    },
+    }),
     transparent: {
         opacity: 0,
     },
     hovered: {
         opacity: 0.5,
     },
-}));
+};
 
 // Shows an arrow pointing to smaller value when sorting is active.
 // signedRank of 0 means no sorting, we only show the arrow on hovering of the header,
 // in the same direction as it will get if clicked (once).
 // signedRank > 0 means sorted by ascending value from lower indices to higher indices
 // so lesser values are at top, so the upward arrow
 const SortButton = (props) => {
-    const classes = useStyles();
     const sortRank = Math.abs(props.signedRank);
-    const visibilityClass =
-        !props.signedRank &&
-        (props.headerHovered ? classes.hovered : classes.transparent);
+    const visibilityStyle =
+        (!props.signedRank || undefined) &&
+        (props.headerHovered ? styles.hovered : styles.transparent);
     return (
-        <div className={clsx(classes.sortDiv)} onClick={props.onClick}>
+        <Box sx={styles.sortDiv} onClick={props.onClick}>
             {props.signedRank >= 0 ? (
-                <ArrowUpwardIcon className={clsx(visibilityClass)} />
+                <ArrowUpwardIcon sx={visibilityStyle} />
             ) : (
-                <ArrowDownwardIcon className={clsx(visibilityClass)} />
+                <ArrowDownwardIcon sx={visibilityStyle} />
             )}
             {sortRank > 1 && !props.hovered && <sub>{sortRank}</sub>}
-        </div>
+        </Box>
     );
 };
 
 const FilterButton = (props) => {
-    const classes = useStyles();
-    const visibilityClass =
+    const visibilityStyle =
         !props.filterLevel &&
-        (props.headerHovered ? classes.hovered : classes.transparent);
+        (props.headerHovered ? styles.hovered : styles.transparent);
```

Comment:
didn't do it because it's not necessary in line 80 wheareas it's necessary at line 63... because line 80 uses mergeSx because we want to combine multiple rules, but line 63 only has the visibility rules

Reply:
But it could be done